packages/ui/lib/components/MiniPlayer/MiniPlayOptions/index.tsx

```diff
@@ -60,7 +60,9 @@ const MiniPlayOptions: React.FC<MiniPlayOptionsProps> = ({
         />
       </button>
       {
-        isExpanded && playOptions.map(playOption => <MiniPlayOptionControl {...playOption} />)
+        isExpanded && playOptions.filter(playOption => {
+          return playOption.dataTestId !== 'mini-player-play-option';
```

Comment:
Hm, this isn't the best way to filter it out, because it should only be used for tests - maybe it should be filtered out at the source?